variables.tf

```diff
@@ -1,23 +1,23 @@
 variable "ibmcloud_api_key" {
-  description = "Used with the Terraform IBM-Cloud/ibm provider."
+  description = "The API key that's used with the IBM Cloud Terraform and IBM provider."
   sensitive   = true
   type        = string
 }
 
 variable "watsonx_admin_api_key" {
   default     = null
-  description = "Used to call Watson APIs to configure the user and the project."
+  description = "The API key that's used to call Watson APIs to configure the user and the project."
```

Comment:
We should provide a more user friendly description. Remove reference to an api call. Something like:  
`The API key of the IBM watsonx admin in the target account`